configure.sh

```diff
@@ -17,57 +17,101 @@ PRIVATE_PUSHWOOSH_PROPERTIES="$BASE_PATH/android/pushwoosh.properties"
 PRIVATE_LIBNOTIFY_PROPERTIES="$BASE_PATH/android/libnotify.properties"
 PRIVATE_NETWORK_CONFIG="$BASE_PATH/android/res/xml/network_security_config.xml"
 SAVED_PRIVATE_REPO_FILE="$BASE_PATH/.private_repository_url"
+SAVED_PRIVATE_BRANCH_FILE="$BASE_PATH/.private_repository_branch"
 TMP_REPO_DIR="$BASE_PATH/.tmp.private.repo"
 
-# TODO: Remove these lines when XCode project is finally generated by CMake. 
-cd $BASE_PATH/3party/boost/
-./bootstrap.sh
-./b2 headers
-cd $BASE_PATH
-
-if [ -f "$SAVED_PRIVATE_REPO_FILE" ]; then
-  PRIVATE_REPO=`cat "$SAVED_PRIVATE_REPO_FILE"`
-  echo "Using stored private repository URL: $PRIVATE_REPO"
-else
-  echo "If you are developer from MAPS.ME team, please specify a private repository url here."
-  echo "If not [yet :)], then just press Enter."
-  echo -n "> "
-  read PRIVATE_REPO
-  if [ -z "$PRIVATE_REPO" ]; then
-    echo "Initializing repository with default values in Open-Source mode."
-    cat "$DEFAULT_PRIVATE_HEADER" > "$PRIVATE_HEADER"
-    cat "$DEFAULT_PRIVATE_CAR_MODEL_COEFS" > "$PRIVATE_CAR_MODEL_COEFS"
-    echo 'ext {
+setup_opensource() {
+  echo "Initializing repository with default values in Open-Source mode."
```

Comment:
Maybe an analogous message for the private mode?

Reply:
Added message in setup_private()